mono/utils/mono-stdlib.c

```diff
@@ -45,7 +50,11 @@ mono_mkstemp (char *templ)
 			return -1;
 		}
 
-		ret = open (templ, O_RDWR | O_BINARY | O_CREAT | O_EXCL, 0600);
+#if HOST_WIN32
+		ret = _open (templ, O_RDWR | O_BINARY | O_CREAT | O_EXCL, 0600);
```

Comment:
Update `g_open`?

Reply:
I can do this.

Reply:
I've fixed this at 4d8c6a42d0f0fc3b377e688ea693ba7985565e9b.